test/test_congress.py

```diff
@@ -14,30 +16,31 @@ def client():
 
 @responses.activate
 def test_all_members_getter(congress, client):
-    # Init testing values.
-    congress_no = '114'
+    congress_no = '115'
     chamber = 'senate'
-    n_members = 100
+    # Read json sample.
+    json_sample_name = 'test/api_output_samples/all_members_sample.json'
```

Comment:
it seems like this logic to load a sample json might be used all over the place, lets abstract this into a function or perhaps even a new `fixture` (this second way can get complicated, but the idea is to have your fixture object a function which loads json files.